examples/minimal/packages/client/src/mud/contractComponents.ts

```diff
@@ -0,0 +1,24 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
this is still autogenerated from `contracts/package.json/tsgen` - I guess for now we should update the `tsgen` script to generate to both the vanilla and react client. I'm hoping we won't need this anymore once #641 lands.